### PR TITLE
Add Yealink T30 T31 T33

### PR DESCRIPTION
**Pull request by andrea-marchionni**
Add new Yealink phones:

- T30/P no line keys
- T31/P/G 2 line keys
- T33P/G 12 line keys, screensaver and wallpaper

nethesis/dev#5862

### CODE REVIEW

File: data/scopes/yealink-T30.ini [hunk 1]
```diff
@@ -0,0 +1,51 @@
+[metadata]
+scopeType = "model"
+displayName = "Yealink T30/P"
```
Comment (Stell0) [inline data/scopes/yealink-T30.ini:3]:
```suggestion
displayName = "Yealink T30P"
```

File: data/scopes/yealink-T30.ini [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+cap_ldap_tls_blacklist =
+provisioning_url_filename = 
+tmpl_phone = "yealink.tmpl"
+cap_linekey_count = 0
+cap_linekey_type_blacklist = ""
+cap_softkey_count = 4
+cap_softkey_type_blacklist = ""
+cap_expkey_count = 0
+cap_expkey_type_blacklist = 
+cap_ringtone_count = 4
```
Comment (Stell0) [inline data/scopes/yealink-T30.ini:26]: are you sure that only 4 ring tones are supported?
```suggestion
cap_ringtone_count = 10
```

File: data/scopes/yealink-T31.ini [hunk 3]
```diff
@@ -0,0 +1,51 @@
+[metadata]
+scopeType = "model"
+displayName = "Yealink T31/G/P"
```
Comment (Stell0) [inline data/scopes/yealink-T31.ini:3]:
```suggestion
displayName = "Yealink T31G/P"
```

File: data/scopes/yealink-T31.ini [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+cap_ldap_tls_blacklist =
+provisioning_url_filename = 
+tmpl_phone = "yealink.tmpl"
+cap_linekey_count = 2
+cap_linekey_type_blacklist = ""
+cap_softkey_count = 4
+cap_softkey_type_blacklist = ""
+cap_expkey_count = 0
+cap_expkey_type_blacklist = 
+cap_ringtone_count = 4
```
Comment (Stell0) [inline data/scopes/yealink-T31.ini:26]:
```suggestion
cap_ringtone_count = 10
```

File: data/scopes/yealink-T30.ini (hunk 2 above)
Comment (andrea-marchionni) [reply data/scopes/yealink-T30.ini:26]: This phone has only 4 ring type

File: data/scopes/yealink-T31.ini (hunk 4 above)
Comment (andrea-marchionni) [reply data/scopes/yealink-T31.ini:26]: This phone has only 4 ring type

**Pull request merged**